src/screens/Customers/components/Record.tsx

```diff
@@ -26,7 +27,7 @@ function Record(props: RecordProps) {
   return (
     <div className={classes.root}>
       <Typography variant="h2" className={classes.unbolded}>
```

Comment:
NIT: In this PR could you also make the typography variant `body2` and the color `secondary` and remove the `unbolded` class? I think that'll make it match the Figma!